CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # App Center SDK for React Native Change Log
 
+## Version 2.0.0
+
+Version 2.0.0 has a **breaking change**, it only supports Xcode 10.0.0+.
+
+### AppCenter Crashes
+
+#### iOS
+
+* **[Feature]** After calling [MSAuth signInWithCompletionHandler:], the next crashes are associated with an accountId corresponding to the signed in user. This is a different field than the userId set by [MSAppCenter setUserId:]. Calling [MSAuth signOut] stops the accountId association for the next crashes.
+* **[Fix]** Print an error and return immediately when calling [MSCrashes notifyWithUserConfirmation:] with confirmation handlers not implemented.
+
+#### Android
+
+* **[Feature]** After calling Auth.signIn, the next crashes are associated with an accountId corresponding to the signed in user. This is a different field than the userId set by AppCenter.setUserId. Calling Auth.signOut stops the accountId association for the next crashes.
+
+### AppCenter Push
+
+#### iOS
+
+* **[Feature]** After calling [MSAuth signInWithCompletionHandler:], the push installation is associated to the signed in user with an accountId and can be pushed by using the accountId audience. This is a different field than the userId set by [MSAppCenter setUserId:]. The push installation is also updated on calling [MSAuth signOut] to stop the association.
```

Comment:
Same as my previous comment.

Let's remove this paragraph and anything related to Auth feature.

Reply:
Removed all Auth feature mentions